portfolio/src/main/webapp/script.js

```diff
@@ -25,26 +25,73 @@ function changeTextBack(text) {
   display.innerHTML = text;
 }
 
+//Form Text Input Resizing Function - Name
+let elName = document.querySelector(".input-wrap-name .input");
+let extraWidthName = document.querySelector(".input-wrap-name .extra-width-name");
+elName.addEventListener("keyup", () => {
+  extraWidthName.innerHTML = elName.value;
+});
+
+//Form Text Input Resizing Function - Message
+let elMessage = document.querySelector(".input-wrap-message .input");
+let extraWidthMessage = document.querySelector('.input-wrap-message .extra-width-message');
+elMessage.addEventListener("keyup", () => {
+  extraWidthMessage.innerHTML = elMessage.value;
+});
+
 function loadComments() {
-  fetch('/list-comments').then(response => response.json()).then((comments) => {
+  var numComments = document.getElementById("num-comments").value;
+
+  fetch('/list-comments?num=' + numComments).then(response => response.json()).then((comments) => {
     const commentListElement = document.getElementById('comments-list');
+    document.getElementById('comments-list').innerHTML = "";
     comments.forEach((comment) => {
       commentListElement.appendChild(createCommentElement(comment));
     })
   });
 }
 
 function createCommentElement(comment) {
-  const commentElement = document.createElement('div');
+  const commentElement = document.createElement('li');
   commentElement.className = 'comment-element';
 
-  const authorElement = document.createElement('h3');
-  authorElement.innerText = comment.author;
+  const authorElement = document.createTextNode(comment.author);
+
+  const messageElement = document.createTextNode(comment.message);
+
+  const spanOne = document.createElement('span');
```

Comment:
Please use meaningful variable names. What do these represent?

Reply:
They represent the spans for the different colors. I changed them to be more meaningful